app/Http/Controllers/DatesController.php

```diff
@@ -13,35 +14,35 @@ class DatesController extends Controller
     public function list() {
         $employeesKochen = Employee::where('gekocht', false)->orderByRaw("SUBSTRING_INDEX(name, ' ', -1)")->get();
         $employeesPraesentieren = Employee::where('praesentiert', false)->orderByRaw("SUBSTRING_INDEX(name, ' ', -1)")->get();
-        $dates = Date::all();
-    
+        $dates = Date::orderBy(DB::raw("STR_TO_DATE(date, '%d.%m.%Y')"), 'desc')->simplePaginate(5);
```

Comment:
Why do we need STR_TO_DATE ?

isn't Date:orderByDesc('date')->simplePaginate(5); also possible?

Reply:
ok then please try to modify the date column to type date and check if it fixes the sorting:
https://laravel.com/docs/11.x/migrations#modifying-columns